cmd/lifecycle/main.go

```diff
@@ -154,7 +156,9 @@ func verifyReadWriteAccess(imageRef string, keychain authn.Keychain) error {
 		return nil
 	}
 	img, _ := remote.NewImage(imageRef, keychain)
-	if !img.CheckReadWriteAccess() {
+	canReadWrite, err := img.CheckReadWriteAccess()
+	if !canReadWrite {
+		cmd.DefaultLogger.Debugf("Error checking read/write access: %s", err)
```

Comment:
We should log that without using Debug as some users using Tekton or kpack could not be aware about how to enable log level !!